pkg/record/recorder.go

```diff
@@ -53,22 +53,22 @@ type recorder struct {
 
 // Event constructs an event from the given information and puts it in the queue for sending.
 func (r recorder) Event(object runtime.Object, reason, message string) {
-	r.EventRecorder.Event(object, corev1.EventTypeNormal, strings.Title(reason), message)
+	r.EventRecorder.Event(object, corev1.EventTypeNormal, cases.Title(language.English).String(reason), message)
```

Comment:
This change changes the output.

Before: `Normal CreateSuccess Create success`
After: `Normal Createsuccess Create success`

Reply:
We can keep the existing behavior by using `cases.NoLower`:

```diff
diff --git a/pkg/record/recorder.go b/pkg/record/recorder.go
index c10b0819..d71abbdf 100644
--- a/pkg/record/recorder.go
+++ b/pkg/record/recorder.go
@@ -53,22 +53,22 @@ type recorder struct {

 // Event constructs an event from the given information and puts it in the queue for sending.
 func (r recorder) Event(object runtime.Object, reason, message string) {
-       r.EventRecorder.Event(object, corev1.EventTypeNormal, cases.Title(language.English).String(reason), message)
+       r.EventRecorder.Event(object, corev1.EventTypeNormal, cases.Title(language.English, cases.NoLower).String(reason), message)
 }
```

**However**, I'm not actually sure what the correct behavior is. @akutz since you committed this code originally, do you have an idea which of the two options is correct?

For reference, these two CAPI providers *don't* use `cases.NoLower`:

- https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1184/files
- https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2425/files

...however CAPA *does*:

- https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3340/files

Additionally, [this](https://github.com/helm/helm/issues/11275) Helm issue addresses the same problem by using `cases.NoLower`.

Looking at the above, I'm leaning towards using `cases.NoLower` since my assumption is that the projects which didn't do so weren't aware of the output difference due to the fact that they don't have a test file for `recorder.go`. Fortunately, CAPV does! 🙂